skops/card/_model_card.py

```diff
@@ -366,6 +368,33 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances) -> "Card":
+        """Visualize permutation importance.
```

Comment:
I think in terms of API, it would make sense to accept importances in a certain way, like accepting both a dict and a pandas dataframe, and then add it to the modelcard.

It doesn't have to be permutation importance.